src/unix/luv/dune

```diff
@@ -0,0 +1,17 @@
+(* -*- tuareg -*- *)
+
+let preprocess =
+   match Sys.getenv "BISECT_ENABLE" with
```

Comment:
This can be separate from this PR, but dune now has direct support for bisect_ppx: https://dune.readthedocs.io/en/latest/instrumentation.html

So the need for an environment variable can be removed from all the dune files and replaced with the direct instrumentation support.

Reply:
This would mean lifting the version required to dune 2.7 (from 1.8), which means dropping ocaml versions from 4.02 to 4.08.

A new PR would be better do discuss the tradeoffs.

Reply:
Earlier versions of OCaml should work just fine with modern dune. There is a ocaml-secondary-compiler package in opam which builds the `dune` binary, and the dune-* libraries are 4.03+

Reply:
The issue is that we force users of Lwt to also raise their Dune version, which might be too aggressive at this point. I agree that this should be discussed separately.